worldedit-core/src/main/java/com/sk89q/worldedit/command/UtilityCommands.java

```diff
@@ -101,6 +101,7 @@ public int fill(Actor actor, LocalSession session, EditSession editSession,
         radius = Math.max(1, radius);
         we.checkMaxRadius(radius);
         depth = Math.max(1, depth);
+        we.checkMaxRadius(depth);
```

Comment:
So from looking at this, it doesn't actually appear that any of the changes in UtilityCommands.java are actually required here. And these do modify the behaviour of the commands.

Can this PR please be modified to only include the changes within EditSession.java?

Reply:
Yeah, will do

Reply:
All done!